e2e_test/e2e_test.go

```diff
@@ -468,6 +507,50 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+
+		g.Context("With more ServiceFunctionChains than resources", g.Ordered, func() {
+			g.It("Should have all SFC Pods eventually running", func() {
```

Comment:
If there is more SFCs then resources then it is not expected that SFC Pods would be eventually running. A subset would be running but the remainder would be in pending state.

Reply:
"eventually" indicates that they all run before the end of the test. It does not mean, that they all run at the same time, which they cannot and do not.

But sure. What would be a better name?

Reply:
reworded